README.md

```diff
@@ -48,19 +48,15 @@ Alternatively to set it up not using Docker use below. This hosts it on `localho
 
 `yarn && yarn cypress `
 
+---
+
 ## Run Cypress tests for M1 chipset
 
-`yarn cypress-for-m1-build` <br>
```

Comment:
Are the deleted commands no longer valid?

Reply:
The deleted commands still work, to run cypress headless in the command line, but I'm not sure the case that a dev would specifically need the command line version if the GUI version here is working - I can leave these headless commands in just to be safe, if need be?